.github/workflows/mac_python_matrix_test.yml

```diff
@@ -0,0 +1,50 @@
+name: MacOS Pyomo Tests Workflow (Slim, Matrixed, Nightly)
+
+on:
+  pull_request:
+    branches:
+      - master
+      # Can add additional branches if desired
+
+jobs:
+  pyomo-mac-tests:
+    name: Pyomo Test (${{ matrix.python-version }}, MacOS)
+    runs-on: macos-latest
+    strategy:
+      max-parallel: 4
+      matrix:
+        python-version: [2.7, 3.5, 3.6, 3.7, 3.8] # All available Python versions
+
+    steps:
+    - uses: actions/checkout@v1 # Checkout branch(es)
+    - name: Set up Python ${{ matrix.python-version }} # Initialize Python version
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install Pyomo dependencies
+      run: |
+        python -m pip install --upgrade pip
+        git clone --quiet https://github.com/Pyomo/pyomo-model-libraries.git
+        pip install --quiet git+https://github.com/PyUtilib/pyutilib
+        python setup.py develop # Install Pyomo
+
+    - name: Install Python modules and Pyomo extensions
+      run: |
+
+        brew update # Install pre-dependencies for pyodbc
+        brew install bash gcc
+        brew link --overwrite gcc
+        brew install pkg-config
+        brew install unixodbc
+        brew install freetds # Now install Python modules
+
+        pip install cython numpy scipy ipython openpyxl sympy pyyaml pyodbc networkx xlrd pandas matplotlib dill seaborn pymysql pyro4 pint pathos
+        
+        pyomo download-extensions # Get Pyomo extensions 
+        pyomo build-extensions
+    - name: Run nightly, not fragile tests with test.pyomo
+      run: |
+        pip install nose
+        KEY_JOB=1
+        test.pyomo -v --cat="nightly" pyomo `pwd`/pyomo-model-libraries # Run nightly, stable tests
+        
```

Comment:
Double check the last line, looks like you might have an invalid character?

Reply:
Fixed.